egg/core/language_analysis.py

```diff
@@ -223,6 +248,7 @@ def posdis(attributes, messages):
         >>> round(PosDisent.posdis(attributes, messages), 6)
         0.978656
         """
+        return gap_mi_first_second(attributes, messages)
         gaps = torch.zeros(messages.size(1))
```

Comment:
Everything below in this method should be deleted?

Reply:
Ops, you're right, forgot to delete